### ISSUE TITLE
Audio file names in KShootMania chart file not trimmed of trailing whitespace

### ISSUE THREAD

**Issue by narcow** [Drewol/unnamed-sdvx-clone#82]
The audio file specification line of KShootMania charts can contain a space between the end of the first file name and the semicolon, e.g. `m=file name.ogg ;file name.ogg`. In unnamed-sdvx-clone, this causes an error specifying that the file cannot be found:

> [Error] Audio file for beatmap does not exists at: "/path/to/file name.ogg "

Altering the chart file to `m=file name.ogg;file name.ogg` (removing the trailing space) causes the chart to play properly, so I believe it should be possible to resolve the issue by trimming whitespace from the audio file names when processing the chart.

**Comment by narcow** [comment]
I have tested only on 64-bit Linux, so it is possible that this issue is platform-specific.

**Comment by Drewol** [comment]
So when I tested this on windows trailing spaces didn't seem to cause an issue but I've added a couple lines to remove any trailing spaces, I haven't verified if it's fixed on linux but I assume it is.

**Comment by narcow** [comment]
Still having the issue, unfortunately. Strange, because the change you made looks like it should have fixed it. Build environment is cmake 3.12.1, GNU make 4.2.1, g++ 8.2.1, and libc++ 4.0.1 (C++11). I'll try to resolve the issue locally and update this issue if I figure it out.

I wrote a simple script to clean ksh files in my /songs directory when I filed this issue, so it's not causing me any trouble currently.
